SPECS/autogen/autogen.spec

```diff
@@ -48,7 +49,7 @@ make DESTDIR=%{buildroot} install
 find %{buildroot} -type f -name "*.la" -delete -print
 
 %check
-make %{?_smp_mflags} check
+make -k check
```

Comment:
Can you add a comment here explaining why we're not using `%{?_smp_mflags}` here?